pax_express_client/utils.py

```diff
@@ -54,13 +64,25 @@ def response_handler(
 
 
 def print_error(message: str):
-    error = typer.style(text=f"{message}", fg=typer.colors.RED, bold=True)
-    typer.echo(f"Error: {error}", err=True)
+    print(f"Error: [red bold] {message}")
 
 
 def print_message(message: str):
-    message = typer.style(text=message, fg=typer.colors.GREEN, bold=True)
-    typer.echo(message)
+    print(f"[green bold] {message}")
+
+
+def print_dict_as_table(data: Dict):
+    table = Table()
+    table.add_column("Field Name", style="cyan")
+    table.add_column("Field Value", style="green")
+    for key in data.keys():
+        table.add_row(key, str(data[key]))
+    print(table)
+
+
+def print_list_as_table(data: List[Dict]):
+    for index in data:
+        print_dict_as_table(data=index)
```

Comment:
it would be better to merge the data into one table. So that it is one table and each entry is one row and not each row is one table